src/gamsproperties.h

```diff
@@ -0,0 +1,42 @@
+#ifndef GAMSARGMANAGER_H
+#define GAMSARGMANAGER_H
+
+#include <QMap>
+#include "file/projectabstractnode.h"
+
+namespace gams {
+namespace studio {
+
+class ProjectGroupNode;
+struct OptionItem;
+
+class GamsProperties
+{
+
+public:
+    GamsProperties(FileId origin);
+
+    void setAndAnalyzeParameters(const QString &inputFile, QList<OptionItem> itemList);
```

Comment:
@rgoltermann violation of the KISS principle. Actually this function does an update. Can we call it `updateParameters` or `updateOptions`?